frontend/components/course-listings/listings.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useSWRAuth } from '@/components/hooks/useSWRAuth';
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Image from 'next/image';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import Container from '@mui/material/Container';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import CardContent from '@mui/material/CardContent';
+import CardHeader from '@mui/material/CardHeader';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import { CourseEvaluationListEntry } from 'utils/api';
+import { Reviews } from '@mui/icons-material';
+import EvaluationList from './evaluationList';
+
+//for now reviews as string but will need to change in reviewer list branch (similar to evaluations)
+type Props = {
+  evaluations: CourseEvaluationListEntry[];
+  reviews: String[];
+};
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+function Listings({ evaluations, reviews }: Props) {
+  const [value, setValue] = React.useState(0);
+
+  const handleChangeTab = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Container
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <Box
+        sx={{ borderBottom: 1, borderColor: 'divider', backgroundColor: '#0E91AC', width: '90%' }}
```

Comment:
again with the background colors... please don't hardcode this... use the `CustomTheme.ts`

see guide in https://mui.com/material-ui/customization/theming/#theme-provider or ask me

Reply:
when I try to use the colours from customTheme on a box element it doesn't work for some reason, I don't know how to go about fixing it

Reply:
here's an example on how to get the theme color (this is my thing from Spontane)
```
import React from "react";
import Box from "@mui/material/Box";
import Fab from "@mui/material/Fab";
**import { useTheme } from "@mui/material/styles";**
import ThumbDownIcon from "@mui/icons-material/ThumbDown";
import ThumbUpIcon from "@mui/icons-material/ThumbUp";

type Props = {
  handleSwipeButtonClick: (liked: Boolean) => () => void;
};

export const ActivitySwiperButton = (props: Props) => {
  **const theme = useTheme();**
  const { handleSwipeButtonClick } = props;
  return (
    <Box
      sx={{
        "& > :not(style)": { m: 1 },
        display: "flex",
        justifyContent: "space-evenly",
        alignItems: "center",
        marginBottom: "12vh",
        width: "100%",
        position: "fixed",
        bottom: 0,
      }}
    >
      <Fab
        sx={{
          backgroundColor: "rgba(255, 255, 255, 0.8);",
          **color: theme.palette.error.main,**
          width: "5rem",
          height: "5rem",
        }}
        onClick={handleSwipeButtonClick(false)}
      >
        <ThumbDownIcon fontSize="large" />
      </Fab>
      <Fab
        sx={{
          backgroundColor: "rgba(255, 255, 255, 0.8);",
          **color: theme.palette.success.main,**
          width: "5rem",
          height: "5rem",
        }}
        onClick={handleSwipeButtonClick(true)}
      >
        <ThumbUpIcon fontSize="large" />
      </Fab>
    </Box>
  );
};
```